crates/pipeline_manager/src/db/pg_setup.rs

```diff
@@ -3,47 +3,50 @@
 //! It caches the downloaded binaries as well. Check the `pg_embed` crate for
 //! more details.
 
-use crate::AnyResult;
-use pg_embed::pg_enums::PgAuthMethod;
-use pg_embed::pg_fetch::{PgFetchSettings, PG_V15};
-use pg_embed::postgres::{PgEmbed, PgSettings};
-use std::path::PathBuf;
+#[cfg(feature = "dev")]
```

Comment:
if you instead make the 

```rust
mod pg_setup;
```

in mod.rs 

as 

```rust
#[cfg(feature = "dev")]
mod pg_setup;
```

you wont have to make an extra module (embedded) for it.